### PR TITLE
chore: update code owners to infra-db-sdk

### PR DESCRIPTION
**Pull request by ttosta-google**
Update sample repo code owners to infra-db-sdk

### CODE REVIEW

File: CODEOWNERS
Comment (jackwotherspoon) [inline CODEOWNERS:43]: @ttosta-google probably need the team to be granted write access first as its showing an error.

Comment (ttosta-google) [reply CODEOWNERS:43]: Thanks @jackwotherspoon for checking it. I confirmed with Eno that we don't have admin access to this repo so the reviewer should fix it after the PR is merged.

Comment (kweinmeister) [reply CODEOWNERS:43]: I've provided write access to your team, so I think we're good to go now?

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**
